cura/Settings/MachineManager.py

```diff
@@ -622,12 +622,7 @@ def activeIntentCategory(self) -> str:
 
         if not global_container_stack:
             return ""
-        intent_category = "default"
```

Comment:
This change is breaking one of the tests. So either this change is wrong, or the test is wrong.

Reply:
Fixed by rebasing on the latest `feature_intent` branch.

Reply:
Confirmed: no more failing tests